src/aig-common/modules/standard/components/social-action/social-action-new-update-form/social-action-new-update-form.component.ts

```diff
@@ -0,0 +1,81 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { FormGroup, FormBuilder, Validators } from '@angular/forms';
+import { MatSnackBar } from '@angular/material/snack-bar';
+import { FuseProgressBarService } from '@fuse/components/progress-bar/progress-bar.service';
+import { SocialActionDTO, SocialActionResourceService } from 'aig-standard';
+import { EventService } from 'aig-common/event-manager/event.service';
+
+@Component({
+    selector: 'aig-social-action-new-update-form',
+    templateUrl: './social-action-new-update-form.component.html',
+    styleUrls: ['./social-action-new-update-form.component.scss']
+})
+export class AigSocialActionNewUpdateFormComponent implements OnInit {
+    step: any = {
+        form: true,
+        loading: false,
+        complete: false
+    };
+
+    constructor(
+        private _formBuilder: FormBuilder,
+        private _fuseProgressBarService: FuseProgressBarService,
+        private _snackBar: MatSnackBar,
+        private socialActionResourceService: SocialActionResourceService,
+        private eventService: EventService,
+    ) { }
+
+    @Input()
+    socialAction: SocialActionDTO;
+
+    socialActionNewUpdateForm: FormGroup;
+
+    ngOnInit(): void {
+        this.socialActionNewUpdateForm = this._formBuilder.group({
+            id:[''],
+            name: ['', Validators.required],
+            code: ['', Validators.required],
+            wikiCode:['']
+        })
+    }
```

Comment:
precompilation?

Reply:
```
        if (this.socialAction != null) {
            this.socialActionNewUpdateForm.patchValue(this.socialAction);
        }
```

Reply:
you have deleted the form definition:
```
        this.socialActionNewUpdateForm = this._formBuilder.group({
            id:[''],
            name: ['', Validators.required],
            code: ['', Validators.required],
            wikiCode:['']
        })
```

Reply:
but i not see it.

Reply:
I already added and pushed through "feature/denis" branch.